normandy/recipes/tests/test_serializers.py

```diff
@@ -28,6 +31,46 @@ def test_it_works(self, rf):
             'is_approved': recipe.is_approved
         }
 
+    def test_it_validates_arguments(self):
```

Comment:
You could refactor it to have another function that takes the data and verifies the assertions, but since it's only 3 sets of data I think that would be overkill. This looks good to me.